### ISSUE TITLE
syntax error when transfer data from sqlite to psql

### ISSUE THREAD

**Issue by TaoZhang5023** [dimitri/pgloader#673]
There is a syntax error when table name is "reference", if the table name is changed then it will be fine.

Here is the debug info:
```
sb-impl::*default-external-format* :UTF-8
tmpdir: #P"/var/folders/r9/5bs4hvr50cz80w9bcrrv6gdh0000gn/T/"
KABOOM!
FATAL error: Could not prepare an sqlite statement.
Code ERROR: near "references": syntax error.
Database: /Users/zhangtao/tenant-options/db/staging.sqlite3
SQL: PRAGMA table_info(references)
Date/time: 2017-11-16-21:26An unhandled error condition has been signalled:
                              Could not prepare an sqlite statement.
Code ERROR: near "references": syntax error.
Database: /Users/zhangtao/tenant-options/db/staging.sqlite3
SQL: PRAGMA table_info(references)


Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {1004D26B33}>
0: (TRIVIAL-BACKTRACE:PRINT-BACKTRACE-TO-STREAM #<SB-IMPL::STRING-OUTPUT-STREAM {1005DF0333}>)
1: (TRIVIAL-BACKTRACE:PRINT-BACKTRACE #<SQLITE:SQLITE-ERROR "Could not prepare an sqlite statement." {1005DF0163}> :OUTPUT NIL :IF-EXISTS :APPEND :VERBOSE NIL)
2: ((FLET "H0" :IN PGLOADER::MAIN) #<SQLITE:SQLITE-ERROR "Could not prepare an sqlite statement." {1005DF0163}>)
3: (SB-KERNEL::%SIGNAL #<SQLITE:SQLITE-ERROR "Could not prepare an sqlite statement." {1005DF0163}>)
4: (ERROR SQLITE:SQLITE-ERROR :FORMAT-CONTROL "Could not prepare an sqlite statement." :FORMAT-ARGUMENTS NIL :DB-HANDLE #<SQLITE:SQLITE-HANDLE {1005A7D263}> :ERROR-CODE :ERROR :ERROR-MSG "near \"references\": syntax error" :STATEMENT NIL :SQL "PRAGMA table_info(references)")
5: (SQLITE:SQLITE-ERROR :ERROR "Could not prepare an sqlite statement." :STATEMENT NIL :DB-HANDLE #<SQLITE:SQLITE-HANDLE {1005A7D263}> :SQL-TEXT "PRAGMA table_info(references)")
6: ((:METHOD INITIALIZE-INSTANCE :AFTER (SQLITE:SQLITE-STATEMENT)) #<SQLITE:SQLITE-STATEMENT {1005DEFF23}>) [fast-method]
7: ((LAMBDA (SB-PCL::|.P0.| SB-PCL::|.P1.|)) #<unavailable argument> #<unavailable argument>)
8: (SQLITE:PREPARE-STATEMENT #<SQLITE:SQLITE-HANDLE {1005A7D263}> "PRAGMA table_info(references)")
9: (SQLITE:EXECUTE-TO-LIST #<SQLITE:SQLITE-HANDLE {1005A7D263}> "PRAGMA table_info(references)")
10: (SQLITE:EXECUTE-TO-LIST #<SQLITE:SQLITE-HANDLE {1005A7D263}> "PRAGMA table_info(references)") [more]
```
Wondering is it due to on line 80 for sqlite-schema.lisp
```
         (sql        (format nil "PRAGMA table_info(`~a`)" table-name)))
```
Should be
```
         (sql        (format nil "PRAGMA table_info('~a')" table-name)))
```
Thanks!

**Comment by dimitri** [comment]
Which version of pglaoder are you using? I can't reproduce the bug here...

**Comment by TaoZhang5023** [comment]
Here is my version:
```
pgloader --version
pgloader version "3.4.1"
compiled with SBCL 1.3.21
```

**Comment by dimitri** [comment]
Can you consider building from source, and/or sharing a SQLite database to reproduce the problem?

I added a table named “reference” to my SQLite test case but failed to reproduce the bug...

```
             table name     errors       rows      bytes      total time
-----------------------  ---------  ---------  ---------  --------------
                  fetch          0          0                     0.000s
            before load          0          2                     0.052s
        fetch meta data          0         26                     0.022s
         Create Schemas          0          0                     0.001s
       Create SQL Types          0          0                     0.000s
          Create tables          0         34                     0.122s
         Set Table OIDs          0         17                     0.002s
-----------------------  ---------  ---------  ---------  --------------
                   long          0          2     0.0 kB          0.054s
                  blobs          0          1    56.6 kB          0.085s
               unsigned          0          2     0.0 kB          0.045s
                  reals          0          3     0.0 kB          0.051s
                   ints          0          3     0.0 kB          0.042s
                    def          0          2     0.0 kB          0.035s
                 stamps          0          3     0.0 kB          0.031s
              character          0          4     0.0 kB          0.022s
                   item          0          2     0.0 kB          0.020s
                 "user"          0          4     0.0 kB          0.026s
                autoinc          0          3     0.0 kB          0.022s
           appointments          0          0                     0.020s
              typenames          0          0                     0.017s
                 intege          2          1     0.0 kB          0.029s
            currentdate          0          1     0.0 kB          0.025s
             table_name          0          0                     0.018s
              reference          0          0                     0.020s
-----------------------  ---------  ---------  ---------  --------------
COPY Threads Completion          0          4                     0.085s
         Create Indexes          0          9                     0.052s
 Index Build Completion          0          9                     0.079s
        Reset Sequences          0          0                     0.037s
           Primary Keys          1          8                     0.016s
    Create Foreign Keys          0          0                     0.000s
        Create Triggers          0          0                     0.000s
       Install Comments          0          0                     0.000s
-----------------------  ---------  ---------  ---------  --------------
      Total import time          2         31    56.8 kB          0.269s
```

**Comment by TaoZhang5023** [comment]
Could you try "references"? Thanks!

**Comment by dimitri** [comment]
Sure:

```
2017-11-22T01:20:44.397166+01:00 LOG Migrating from #<SQLITE-CONNECTION sqlite:///...pgloader/test/sqlite/sqlite.db #x3020071459AD>
2017-11-22T01:20:44.398745+01:00 LOG Migrating into #<PGSQL-CONNECTION pgsql://dim@UNIX:5432/pgloader #x3020071453CD>
2017-11-22T01:20:45.311252+01:00 ERROR PostgreSQL ["intege"] Database error 22P02: invalid input syntax for integer: "toto"
CONTEXT: COPY intege, line 2, column id: "toto"
2017-11-22T01:20:45.329131+01:00 ERROR PostgreSQL ["intege"] Database error 22P02: invalid input syntax for integer: "12.3D0"
CONTEXT: COPY intege, line 1, column id: "12.3D0"
2017-11-22T01:20:45.503333+01:00 ERROR PostgreSQL Database error 23502: column "id" contains null values
QUERY: ALTER TABLE autoinc ADD PRIMARY KEY USING INDEX idx_129892_sqlite_autoindex_autoinc_1;
2017-11-22T01:20:45.512439+01:00 LOG report summary reset
             table name     errors       rows      bytes      total time
-----------------------  ---------  ---------  ---------  --------------
                  fetch          0          0                     0.000s
            before load          0          2                     0.121s
        fetch meta data          0         26                     0.027s
         Create Schemas          0          0                     0.002s
       Create SQL Types          0          0                     0.001s
          Create tables          0         34                     0.107s
         Set Table OIDs          0         17                     0.004s
-----------------------  ---------  ---------  ---------  --------------
                   long          0          2     0.0 kB          0.598s
                  blobs          0          1    56.6 kB          0.653s
               unsigned          0          2     0.0 kB          0.586s
                  reals          0          3     0.0 kB          0.571s
                   ints          0          3     0.0 kB          0.044s
                    def          0          2     0.0 kB          0.036s
                 stamps          0          3     0.0 kB          0.032s
              character          0          4     0.0 kB          0.068s
                   item          0          2     0.0 kB          0.030s
                 "user"          0          4     0.0 kB          0.029s
                autoinc          0          3     0.0 kB          0.037s
           appointments          0          0                     0.021s
              typenames          0          0                     0.017s
                 intege          2          1     0.0 kB          0.022s
            currentdate          0          1     0.0 kB          0.026s
             table_name          0          0                     0.027s
           "references"          0          0                     0.026s
-----------------------  ---------  ---------  ---------  --------------
COPY Threads Completion          0          4                     0.071s
         Create Indexes          0          9                     0.049s
 Index Build Completion          0          9                     0.097s
        Reset Sequences          0          0                     0.051s
           Primary Keys          1          8                     0.016s
    Create Foreign Keys          0          0                     0.000s
        Create Triggers          0          0                     0.000s
       Install Comments          0          0                     0.000s
-----------------------  ---------  ---------  ---------  --------------
      Total import time          2         31    56.8 kB          0.285s
NIL
```

**Comment by TaoZhang5023** [comment]
Thanks! I think there might be some error on my side. Currently I have to change the table name to load it to psql and then change the name back...

**Comment by dimitri** [comment]
Did you try compiling pgloader from fresh sources? it might be a bug that has been fixed after 3.4.1.
